x11/pango/Portfile

```diff
@@ -59,7 +54,6 @@ configure.args-append \
                         -Dfreetype=enabled \
                         -Dintrospection=enabled \
                         -Dlibthai=disabled \
-                        -Dinstall-tests=false \
```

Comment:
Lukas, why are you removing this?

Reply:
I can't remember for sure, but I don't think it's available as an option anymore, so I had to remove it.

Reply:
> I can't remember for sure, but I don't think it's available as an option anymore, so I had to remove it.

It was renamed to `build-testsuite`, for consistency with `build-examples`. Just pushed a fix.